core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -54,7 +54,7 @@
     private Map<String, Icon> iconsByClassSpec = new ConcurrentHashMap<>();
     private Map<String, Icon> coreIcons = new ConcurrentHashMap<>();
 
-    private static final String PLACEHOLDER_SVG = "<svg xmlns=\"http://www.w3.org/2000/svg\" class=\"ionicon\" viewBox=\"0 0 512 512\"><title>Ellipse</title><circle cx=\"256\" cy=\"256\" r=\"192\" fill=\"none\" stroke=\"currentColor\" stroke-linecap=\"round\" stroke-linejoin=\"round\" stroke-width=\"32\"/></svg>";
+    private static final String PLACEHOLDER_SVG = "<svg xmlns=\"http://www.w3.org/2000/svg\" class=\"ionicon\" height=\"48\" viewBox=\"0 0 512 512\"><title>Close</title><path fill=\"none\" stroke=\"currentColor\" stroke-linecap=\"round\" stroke-linejoin=\"round\" stroke-width=\"32\" d=\"M368 368L144 144M368 144L144 368\"/></svg>";
```

Comment:
Why is the title "Close"? Is that shown to users and can give the impression of a "Close Window" button?

Reply:
Not necessarily any "x" shown somewhere refers to closing something, e.g. aborting builds.
I thought that it would fit fine here, because it's somewhat of obvious that something went wrong.
However, I can look for a different ionicon for sure :)

Reply:
This is about the title attribute, not the choice of icon.

Reply:
Oh, I read that wrong then. My bad, but no, this is not shown to users.